.mailmap

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# This file maps author and committer email addresses to canonical names and email
+# addresses for display in the output of `git log` and `git blame`.
+# Format:
+# Canonical Name <canonical@address.dns> <actual@address.dns>

```

Comment:
Additionally, I think we should follow the example of https://git-scm.com/docs/gitmailmap and say "commit" instead of "actual" for the second address.